test/unit/util.test.ts

```diff
@@ -60,3 +60,10 @@ test('Encoding ArticleId for Zim HTML Url', async(t) => {
         t.equal(encoded, encodeArticleIdForZimHtmlUrl(unencoded), `encodeArticleIdForZimHtmlUrl() encoding`);
     }
 });
+
+test('wikitext comparison', async(t) => {
+    const wikicode = `According to the principle of conservation of mechanical energy, the mechanical energy of an [[isolated system]] remains constant in time, as long as the system is free of [[friction]] and other non-conservative forces.`;
+    const html = `<p id="mwAQ">According to the principle of conservation of mechanical energy, the mechanical energy of an <a rel="mw:WikiLink" href="./Isolated_system" title="Isolated system" id="mwAg">isolated system</a> remains constant in time, as long as the system is free of <a rel="mw:WikiLink" href="./Friction" title="Friction" id="mwAw">friction</a> and other non-conservative forces.</p>`;
+    const resultHtml = await convertWikicodeToHtml(wikicode, 'https://en.wikipedia.org/');
+    t.equals(html, resultHtml, 'HTML matches')
```

Comment:
please follow the style with the tail

Reply:
@midik I am not quite sure that I understood this correctly.

Reply:
@bakshiutkarsha sorry, I just mean a semicolon at the end of the line 68 ;)

Reply:
Did that change already :)